gwsurrogate/surrogate.py

```diff
@@ -1617,15 +1617,20 @@ def __call__(self, q, chiA0, chiB0, M=None, dist_mpc=None, f_low=None,
                 using times/freqs. Default None.
 
     ellMax:     Maximum ell index for modes to include. All available m
-                indicies for each ell will be included automatically.
-                Default: None, in which case all available modes wll be
-                included.
+                indicies for each ell will be included automatically. The 
+                m<0 modes will automatically be included for nonprecessing
+                models. 
+                Default: None, in which case all available modes will be
+                included. For nonprecessing, setting ellMax=None will
+                not return m<0 modes.
```

Comment:
Do we want to change this behavior? The natural is to return m<0 even for None.

Reply:
The only issue is that some people might already be using the code and expect certain behavior. For example, RIFT ties into gwsurrogate to get the waveform modes. If the default is to now return the m<0 modes this could mess up current workflows.

I agree that in hindsight your suggestion is the natural thing to do. 

I'm not opposed to changing it either. Just wondering how cautious we should be with such changes (maybe we can email people who we already know are using gwsurrogate)